src/EditorFeatures/Core/AddImports/AbstractAddImportsPasteCommandHandler.cs

```diff
@@ -18,7 +16,7 @@
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
 
-namespace Microsoft.CodeAnalysis.Editor.Implementation.AddImports
+namespace Microsoft.CodeAnalysis.AddImport
```

Comment:
```suggestion
namespace Microsoft.CodeAnalysis.AddImports
```

Reply:
This is intentional. We already have AddImport namespace in Features.